website/scripts/control.js

```diff
@@ -107,14 +107,11 @@ function createRadioElement(container, radioId, radioTitle, itemCount, groupName
     newInput.type = 'radio';
     newInput.name = groupName;
 
-    //TODO: Remove this when RouteID has been changed to have no spaces.
-    //Currently, RouteID and RouteName are same and have spaces, but ID should not have any spaces. For now, updating to have underscore
-    var revisedId = radioId.toString().replace(/ /g, '_');

```

Comment:
So Mike after re-reading my comment above, the replace command is to resolve the issue when the filename has spaces. I am using the filename as the ID. So the real fix would be on Route.msg to have RouteID to be numerical or not have spaces, and maybe underscores/dashes/special characters. And have Route Name be the actual file name with underscores/dashses/spaces/special characters (if allowed in Linux filenaming). IDs should be unique so I'd go with numerical value. 

Once that's fixed then your fixes here should work. 

HEre's the route.msg format. Currently route_generator updates routeID and routeName with the filename. 
# The route id
string routeID

# The route name for display to users
string routeName

Reply:
@msmcconnell @qswawrq another thought is to remove routeID in route.msg, and just have routeName be the filename in the new routeNode. This is under the assumption we don't really use routeID anywhere else besides route_generator. 

If you agree that we don't really use RouteID anywhere else, then for now, I would say update the UI to automatically generate or re-generate the radioID using numerical values and have the radioTitle to be the routeFileName (or routeName). Then I don't think you will need to make changes to the route_generator.

Reply:
Nevermind, I see RouteID getting referenced in RouteState.msg and SetActiveRoute.msg, which would impact guidance I assume. So I think just go back to my original suggestion above. Also verified in our old repo, I see comment in the RouteWorker.java in the old repo (line229) saying ".setRouteID(route.getRouteName()); //TODO come up with better method of defining the route id". And that's where my comment in the UI matches that it expects routeID and routeName to be different.